tutor/templates/build/forum/Dockerfile

```diff
@@ -48,5 +48,6 @@ ENV MONGODB_AUTH ""
 ENV MONGOID_AUTH_MECH ""
 ENV MONGODB_HOST "mongodb"
 ENV MONGODB_PORT "27017"
+ENV FORUM_MONGODB_DATABASE "cs_comments_service"
```

Comment:
Do you want to use {{ FORUM_MONGODB_DATABASE }} here?

Reply:
> Do you want to use {{ FORUM_MONGODB_DATABASE }} here?

Actually, no: if we do, the Docker image will depend on the actual Tutor configuration, which is not a good thing. I think it's better if we keep it like this. I would simply rename the environment variable to `MONGODB_DATABASE`: since we are already in the context of the forum, it's not very useful to add the `FORUM_` prefix.

(and don't forget the rename the variable in the other templates, too)

Reply:
Make sense.